README.md

```diff
@@ -5,7 +5,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 # Azure Terraform Module Generator
 `generator-az-terra-module` creates base template to start a new terraform module.
 
-![Alt Text](/doc/scaffolding.640x434.gif)
+![Terraform module template walkthrough](/doc/scaffolding.640x434.gif)
```

Comment:
Shall we update the GIF to include the optional Dockerfile step ?

Reply:
Yes, the `.gif` needs to be updated to include the new option. I will open a task to track this.